pkg/util/objectwatcher/objectwatcher.go

```diff
@@ -126,7 +126,10 @@ func (o *objectWatcherImpl) retainClusterFields(desired, observed *unstructured.
 func (o *objectWatcherImpl) Update(clusterName string, desireObj, clusterObj *unstructured.Unstructured) error {
 	updateAllowed := o.allowUpdate(clusterName, desireObj, clusterObj)
 	if !updateAllowed {
-		return nil
+		// The existing resource is not managed by Karmada, and no conflict resolution found, avoid updating the existing resource by default.
+		return fmt.Errorf("resource(kind=%s, %s/%s) already exist in cluster %v and the %s strategy value is empty, karmada will not manage this resource",
```

Comment:
xxx %s strategy value is empty or invalid.

Reply:
The log comes from https://github.com/karmada-io/karmada/blob/master/pkg/util/objectwatcher/objectwatcher.go#L278